tractify/utils.py

```diff
@@ -193,7 +193,8 @@ def collect_dmripreproc_output(dmriprep_dir, subject_id, session_id = None):
     dmri_output['eddy_file'] = os.path.join(dmri_files_path, 'eddy_corrected.nii.gz')
     dmri_output['bvec'] = os.path.join(dmri_files_path, 'eddy_corrected.eddy_rotated_bvecs')
     dmri_output['bval'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('.bval')))
-    dmri_output['dwi_mask'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('brain_mask.nii.gz')))
+    #dmri_output['eddy_avg_b0'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('eddy_corrected_avg_b0.nii.gz')))
```

Comment:
can you remove this commented out line

Reply:
sure, i also removed the rest of the extra comments